samples/onboarding_manual.ipynb

```diff
@@ -0,0 +1,1519 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Welcome to Synnax Lab! In this tutorial, I'll guide you through the entire process—from setting up your account to making your first submission. Let's go! 🚀"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Creating Account"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step: 1 Navigate to Synnax Lab\n",
+    "\n",
+    "Let's go to [Synnax Lab](https://synnax.app/) and click on the **LOG IN** button.\n",
+    "\n",
+    "\n",
+    "![Synnax](./img/step_1.png)\n",
+    "<!-- ![Synnax](https://github.com/synnax-ai/synnax-lab-sdk/blob/update_manual/samples/img/step_1.png?raw=true) -->"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 2: Choosing Your Login Method\n",
+    "\n",
+    "Synnax allows you to log in with a wallet or an email. Click LOG IN in the top right corner, choose either option and finish the sign up process. This will register you in the Synnax App, which is currently open for everyone. It allows registered users to view the credit intelligence for the companies Synnax features. To become a contributor proceed to Step 3.\n",
```

Comment:
> Click LOG IN in the top right corner
Duplicates step 1

Reply:
Did you reply to the correct comment? Anyway,
<img width="381" alt="image" src="https://github.com/user-attachments/assets/93b0e55c-eaf2-43e1-b8b0-bf874392facd">
These are all file additions, green is file additions. I think you placed them in the wrong folder.